### PR TITLE
Feature/matches backend with some items from VC

### PR DESCRIPTION
**Pull request by wilsonwtcheng**

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+
```
Comment (h8rry) [inline app/controllers/likes_controller.rb]: Please check your indentations in your PR and make sure they are correct before sending it.

File: app/assets/javascripts/angular/controllers/posts/randomitem.js.erb [hunk 2]
```diff
@@ -2,10 +2,21 @@ app.controller('RandomShowCtrl',['$scope', '$http', '$location', function($scope
...
+      newUrl + '/' + $scope.post.id;
+    }
+
+    $http.get(newUrl).success(function(response){
+      $scope.post = response;
+      console.log(response.title)
+    });
+  };
+
+  $scope.cross();
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/posts/randomitem.js.erb:20]: Do you need to use the function `cross()` in the front end? If not, then you don't need to put it inside `$scope`

File: app/assets/javascripts/angular/controllers/navbar.js.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+// app.controller('NavBarCtrl', function($scope){
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/navbar.js.erb]: They are commented out codes

File: app/assets/javascripts/angular/controllers/posts/randomitem.js.erb (hunk 2 above)
Comment (wilsonwtcheng) [inline app/assets/javascripts/angular/controllers/posts/randomitem.js.erb:20]: We do use it in frontend

File: app/assets/javascripts/angular/controllers/navbar.js.erb (hunk 3 above)
Comment (wilsonwtcheng) [inline app/assets/javascripts/angular/controllers/navbar.js.erb]: Will take out file- thanks

File: app/controllers/likes_controller.rb (hunk 1 above)
Comment (wilsonwtcheng) [inline app/controllers/likes_controller.rb]: Have altered default settings and will keep an eye out for this. Thanks

**Comment by h8rry** [comment]
Looks fine

**Pull request merged**